src/shared/api/sessionServiceAPI.ts

```diff
@@ -0,0 +1,59 @@
+import * as request from 'superagent';
+import {getSessionServiceApiUrl} from "./urls";
+import {IVirtualStudy} from "shared/model/VirtualStudy";
+
+
+type VirtualStudyData = {
```

Comment:
No users?

Reply:
not all fields are required in the front-end. Unused and unwanted fields are filtered in the session-service proxy api